net/quic/quic_chromium_packet_reader.cc

```diff
@@ -129,7 +134,7 @@ bool QuicChromiumPacketReader::ProcessMultiplePacketReadResult(int result) {
       continue;
     }
     quic::QuicReceivedPacket packet(read_packet->buffer, read_packet->result,
-                                    clock_->Now());
+                                    clock_->ApproximateNow());
```

Comment:
`#ifdef` here too ?

Reply:
This already is gated, because it's in `ProcessMultiplePacketReadResult`.